Dockerfile

```diff
@@ -1 +1,10 @@
 FROM <BASE>
+
+ENV foo /bar
+
+ARG WSO2_USERNAME
```

Comment:
I think we have to look for another way to pass credentials to the build. 
```
It is not recommended to use build-time variables for passing secrets like github keys, user credentials etc. Build-time variable values are visible to any user of the image with the docker history command.

Refer to the “build images with BuildKit” section to learn about secure ways to use secrets when building images.
````
Reference: https://docs.docker.com/engine/reference/builder/#arg